cadence/activity_loop.py

```diff
@@ -27,69 +28,85 @@ def activity_task_loop(worker):
             if worker.is_stop_requested():
                 return
 
-            # fetch activity task 
-            # TODO: Add back polling time
+            polling_start = datetime.datetime.now()
             try:
-                task, err = service.poll_for_activity_task(polling_request)
-                if err:
-                    logger.error("PollForActivityTask failed: %s", err)
-                    continue
-                # TODO: move this one to handle_task() or check 
-                elif not task.task_token:
-                    logger.debug("PollForActivityTask has no task_token (expected): %s", task)
-                    continue
-                logger.debug("PollForActivityTask: done")
+                task = service.poll_for_activity_task(polling_request)
+            except RuntimeError as ex:
```

Comment:
Maybe it's a good idea to use one catch, does not really matter if it's runtime err or not.

Reply:
Agree

Reply:
Delete this one?

Reply:
should be?